lib/src/callable/async.dart

```diff
@@ -36,7 +36,7 @@ abstract class AsyncCallable {
   /// The argument declaration is parsed from [arguments], which should not
   /// include parentheses. Throws a [SassFormatException] if parsing fails.
   ///
-  /// See [new Callable] for more details.
+  /// See [Callable] for more details.
```

Comment:
This `new` is doing real work here—it indicates that we're trying to link specifically to the default constructor, *not* to the class's general documentation. Is there another recommended way to indicate that?

Reply:
Ah it seems the "new way" is [MyClass.new] or [MyClass.myNamedConstructorOrFactory]